x/simulation/params.go

```diff
@@ -134,7 +134,7 @@ func RandomParams(r *rand.Rand) Params {
 		PastEvidenceFraction:      r.Float64(),
 		NumKeys:                   RandIntBetween(r, 2, 250),
 		EvidenceFraction:          r.Float64(),
-		InitialLivenessWeightings: []int{r.Intn(80), r.Intn(10), r.Intn(10)},
+		InitialLivenessWeightings: []int{RandIntBetween(r, 1, 80), r.Intn(10), r.Intn(10)},
```

Comment:
Would it make sense to "protect" the last two parameters to be above 0 as well? If not, maybe another sampling procedure would be required to insure that the `sum > 0`. What do you think?

Reply:
What I mean by that is that any of these values (single one or two of them) being 0 is valid, what is not is if they are all 0.

Reply:
Nope I don't think it matters, As long as there is "some" liveliness we're good aka "protecting" just one of the weightings is fine. I think it's good to simulate with a liveliness weighting of 0 as it's a valid value.